### PR TITLE
Add internal field to Route proto struct

### PR DESCRIPTION
**Pull request by nhsieh**
We're adding a new field to the proto struct as a prerequisite to a PR for cloud_controller_ng.

[#161350158]

Signed-off-by: Slawek Ligus <sligus@pivotal.io>

### CODE REVIEW

File: api/protos/cloud_controller.proto [hunk 1]
```diff
@@ -38,6 +38,7 @@ message Route {
   string guid = 1;
   string host = 2;
   string path = 3;
+  bool internal = 4;
```
Comment (utako) [inline api/protos/cloud_controller.proto]: where does the internal bool’s default live? will it be defaulted? is it optional?

**Comment by KauzClay** [comment]
@tylerschultz @oozie can you please rebase and rerun the generate protos script using the most recent changes?

File: api/protos/cloud_controller.proto (hunk 1 above)
Comment (moleske) [reply api/protos/cloud_controller.proto]: In proto3 syntax, explicit defaults are not supported. The implicit default is `false`, which is what we want.

**Comment by moleske** [comment]
We have regenerated protos and bumped the ruby client version. @utako @KauzClay how is it looking?

**Pull request merged**